### DISCUSSION TITLE
Does Debug Mode works?

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1256] [Q&A]
A simple case like this:
```cpp
// test.cpp
#include <iostream>

using namespace std;

int main() {
  int a = 0;
  while (1) {
    a++;
    cout << a << endl;  // F8 to toggle breakpoint here.
  }
  return 0;
}
// g++ -g test.cpp -o test
```
`F6` to run `test`，It seems that program is not stopped at F8 position.

**Comment by ayamir** [comment]
Why doesn't？
![image](https://github.com/ayamir/nvimdots/assets/61657399/4b835874-ba9a-4894-a6bf-85ed20599b4c)

**Reply by yinghaoyu** [reply]
My nvim shows:
![Snipaste_2024-05-13_09-02-25](https://github.com/ayamir/nvimdots/assets/42765754/d6a06da4-bbaa-4e69-8733-a7c9bbda92c1)

**Reply by yinghaoyu** [reply]
I don't know any cure about this.

**Reply by ayamir** [reply]
Are you using `nvim` installed on windows to debug a file on wsl?

**Reply by yinghaoyu** [reply]
Both of them are in wsl.

**Comment by ayamir** [comment]
I guess it b/c adapter doesn't attach to the debug process.
